CHANGELOG.md

```diff
@@ -13,6 +13,7 @@
     interstitial ad is ready to be displayed.
   - Provide default implementation in interstitial listener. Java 8 is required, see
   https://developer.android.com/studio/write/java8-support.
+  - Remove the deprecated `Criteo#init` method. `Criteo.Builder#init` should be used instead.
```

Comment:
We could link to the method directly using markdown.
[Criteo.Builder#init](https://github.com/criteo/android-publisher-sdk/blob/main/publisher-sdk/src/main/java/com/criteo/publisher/Criteo.java#L54)

```
\[Criteo.Builder\](https://github.com/criteo/android-publisher-sdk/blob/main/publisher-sdk/src/main/java/com/criteo/publisher/Criteo.java#L54)
```

Reply:
Ok, but I'm not a really big fan of this. Changing the code (just adding/removing an include for instance) will make this link invalid.